<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImpPrivKeyDialog</class>
 <widget class="QDialog" name="ImpPrivKeyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import pakcoin into your wallet</string>
  </property>
  <widget class="QPushButton" name="importButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Import PrivateKey</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/add</normaloff>:/icons/add</iconset>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>415</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::NoButton</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="addrEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>471</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Your private address</string>
   </property>
   <property name="statusTip">
    <string>Your private address</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Key:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>140</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16177186</height>
    </size>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/quit</normaloff>:/icons/quit</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>491</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Note: this operation will rebuild the index data and wait for a long time</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../bitcoin.qrc"/>
 </resources>
 <connections/>
</ui>
